src/viam/ur/module/test.cpp

```diff
@@ -114,18 +114,30 @@ BOOST_AUTO_TEST_CASE(test_write_trajectory_to_file) {
 using namespace std::chrono_literals;
 
 BOOST_AUTO_TEST_CASE(test_waypoints_filename) {
-    auto x = waypoints_filename("/home/user", 1747161493357ms);
-    BOOST_CHECK_EQUAL(x, "/home/user/1747161493357_waypoints.csv");
+    const std::string k_path = "/home/user";
+    const auto timestamp = unix_time_iso8601();
```

Comment:
was getting tidy errors trying to pull these out to constants for the different tests to use. 
kept it as `const std::string` because `const auto` kept trying to make it a char array, which doesn't have the convenience of using the `+` with strings for the path.